src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,3 +1,22 @@
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+import React from 'react';
+
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  if (user instanceof Object) {
```

Comment:
```suggestion
  if (user) {
```
more simple, but has the same effect 😉